core/store/ledgerstore/ledger_store.go

```diff
@@ -502,7 +502,9 @@ func (this *LedgerStoreImp) verifyHeader(header *types.Header) error {
 			for _, p := range blkInfo.NewChainConfig.Peers {
 				peerInfo[p.ID] = p.Index
 			}
+			this.lock.Lock()
 			this.vbftPeerInfoMap[header.Height] = peerInfo
+			this.lock.Unlock()
```

Comment:
this is not enough to avoid concurrent read write access.

Reply:
in read map place add read lock